editor/src/core/workers/parser-printer/parser-printer-parsing.ts

```diff
@@ -2612,6 +2616,26 @@ export function parseOutJSXElements(
       : expression.whenFalse
     const whenFalseBlock = parseClause(innerWhenFalse)
 
+    // We need to create the UID here before parsing expression.condition, because we
+    // might be stealing one specified in the comment attached to the expression.condition
```

Comment:
yikes 😬

Reply:
Actually I'll update this comment a bit. Basically the overrides we're using right now for the conditional type are actually the leading comments of the `expression.condition`, meaning that if we wish to continue doing that then we can't have separate overrides for both the conditional as a whole and the `expression.condition`. That hasn't mattered thus far, but it means right now we can't provide a hard coded UID for the `expression.condition`